satellite/satellitedb/overlaycache.go

```diff
@@ -803,7 +810,7 @@ func convertDBNode(ctx context.Context, info *dbx.Node) (_ *overlay.NodeDossier,
 			Release:    info.Release,
 		},
 		Contained:    info.Contained,
-		Disqualified: info.Disqualified,
+		Disqualified: disqualified,
```

Comment:
Not sure if currently use `*time.Time` for business logic types or use a `time.Time`, which is what it has been used here and due to the DB uses a `*time.Time` it must be mapped accordingly.

On the other hand, there may be some helper function in our current code base for this conversions, but I don't know as I asked in other means and I didn't get an answer at that time and I moved one.

Reply:
I would make everything a *time.Time

Reply:
ok!